### ISSUE TITLE
Error Generating cloglog Prediction in Rstudio from Markdown File

### ISSUE THREAD

**Issue by abfratz25** [wallaceEcoMod/wallace#152]
After downloading my wallace session code as a .Rmd file, I am having trouble reproducing the results of the analysis in Rstudio in order to generate a cloglog prediction. 

Steps to reproduce:
1. Obtain occurrence points in Module 1 from GBIF (n = 100, Eptesicus fuscus)
2. Process occurrence data in Module 2 using Spatial Thin (distance = 10 km)
3. Obtain environmental data in Module 3 selecting WorldClim Bioclims (bio 1-19, 5 arcmin resolution)
4. Process environmental data in Module 4 using minimum convex polygon (1 degree buffers) and 350 background points
5. Partition occurrence data in Module 5 using spatial partition (Block k =4)
6. Build and evaluate models in Module 6 using Maxent (feature: LQHP, regularization multipliers: 1-2, multiplier step value: 1)
7. Visualize the Model results in Module 7 as a map prediction (prediction output: cloglog, threshold: minimum training presence)
8. Download session code as .Rmd and attempt to run in Rstudio

After running the following lines of code:
`# Select your model from the models list`
`mod <- evalMods[["L_1"]]`
`# generate cloglog prediction`
`pred <- dismo::predict(mod, envsBgMsk, args=c("outputformat=cloglog"))`

I get the error:
`Error in UseMethod("predict") : `
`  no applicable method for 'predict' applied to an object of class "NULL" `

It seems that `ENMeval` does not save the HTML files for the models, thus they cannot be used in the subsequent code. Is there a workaround for reproducing and visualizing the wallace results in Rstudio?

System info:
RStudio 1.1.453 on Mac, R version 3.5.1

**Comment by gepinillab** [comment]
Hi @abfratz25,

Thanks for posting, I was able to reproduce your issue. This one is a bug in the names of the evalMods object. You have two options for fixing:

1. Add the `names(evalMods) <- e@results$settings` in the **Build and Evaluate Niche Model** section of your .Rmd file
```R
evalTbl <- e@results
evalMods <- e@models
names(evalMods) <- e@results$settings
evalPreds <- e@predictions
```
or

2. Install a temporal branch that fixed the problem until the issue is fixed in the 'master' branch of wallaceEcoMod/wallace.
```R
library(devtools)
install_github("gepinillab/wallace", ref = "evalModNames", dependencies = F)
```
Please, let me know if some of this option fix your problem.

Best,
Gonzalo

**Comment by abfratz25** [comment]
Hi Gonzalo,

The first option (adding `names(evalMods) <- e@results$settings`) worked perfectly! Thank you!

Best,
Adriana

**Comment by gepinillab** [comment]
Merged in #154
